src/components/PhoneNumberInput.tsx

```diff
@@ -16,7 +16,8 @@ export default class PhoneNumberInput extends React.Component<
   };
 
   onNumberChange = (value: string) => {
-    if (!value.match(/^\d*$/)) {
+    value = value.replace(/\D/g, "");
```

Comment:
please don't override arguments. new variable ?

Reply:
Well stings are immutable so will not change value in calling function. But will do it once I reach home.

Reply:
done